### PR TITLE
Create todo functionalities

### PR DESCRIPTION
**Pull request by Ismailco**
- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- By default new tasks have the property completed set to false.
- By default new tasks have the property index set to the value of the new array length.
- Deleting a task updates all the remaining items indexes

### CODE REVIEW

File: src/functionality.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+};
+
+export const addTask = (desctiption, completed = false, index, todoList) => {
+  const task = {
+    desctiption,
+    completed,
+    index,
+  };
+  todoList.todoTasks.push(task);
+  // displayTasks();
```
Comment (mikethreels) [inline src/functionality.js]: - [x] please remove all the commented out code in order to keep your codebase clean and readable.

File: src/functionality.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  setLocalStore(todoList.todoTasks);
+};
+
+export const removeTask = (i, todoList) => {
+  todoList.todoTasks.splice(i, 1);
+  todoList.todoTasks.forEach((task) => {
+    task.index = i + 1;
+  });
+  // displayTasks();
```
Comment (mikethreels) [inline src/functionality.js]: - [x] please remove all the commented out code in order to keep your codebase clean and readable.

File: src/functionality.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+export const editTask = (input, listItem, i, todoList) => {
+  if (input.readOnly) {
+    input.readOnly = false;
+    listItem.style.backgroundColor = 'rgb(182, 166, 166)';
+  } else {
+    input.readOnly = true;
+    todoList.todoTasks[i].desctiption = input.value;
+    setLocalStore(todoList.todoTasks);
+    // displayTasks();
```
Comment (mikethreels) [inline src/functionality.js]: - [x] please remove all the commented out code in order to keep your codebase clean and readable.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ismail, 

Good job on creating the add and delete functions for the to-do items 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @Ismailco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**